Gopkg.toml

```diff
@@ -31,9 +31,9 @@ ignored = [
     ]
 
 required = [
-    "k8s.io/code-generator/cmd/deepcopy-gen", # for go generate
-    "sigs.k8s.io/controller-tools/cmd/controller-gen", # for crd/rbac generation
-    "golang.org/x/tools/cmd/stringer", # for go generate
```

Comment:
Note that deepcopy-gen and stringer were unused. We use controller-gen to generate deepcopy methods.